man/adoc/bpftrace.adoc

```diff
@@ -1922,7 +1922,26 @@ The non-standard ones can be found in the table below:
 
 |===
 
-Supported escape sequences
+`printf()` can also symbolize enums as strings. For example:
+
+----
+enum custom {
+  CUSTOM_ENUM = 3,
+};
+
+tracepoint:skb:kfree_skb {
+  $r = args->reason;
+  printf("%d, %s, %s\n", $r, $r, CUSTOM_ENUM);
+}
+----
+
+may yield:
```

Comment:
This feels confusing. Why not just use a `BEGIN` probe so you can say what it definitely will print out.

Reply:
Yeah, agreed. Simplified.